app/controllers/account/users_controller.rb

```diff
@@ -0,0 +1,46 @@
+class Account::UsersController < Account::AccountController
+  
+  def index
+    @users = User.all
+  end
+
+  def show
+    @user = User.find(params[:id])
```

Comment:
move ` User.find(params[:id])` to the some private method, and reuse it in show, edit, update, destroy actions.